RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,47 @@
+# React Native Paper Component Templates
+
+This Document contains a set of components Templates around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.
```

Comment:
This Document contains a set of component templates around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.

Reply:
just suggestion see what others say

Reply:
I would not call them templates. More like style overrides.